app/controllers/api/v1/medical_shifts_controller.rb

```diff
@@ -45,6 +45,18 @@ def update
         end
       end
 
+      def destroy
+        authorize(medical_shift)
+
+        result = MedicalShifts::Destroy.result(id: medical_shift.id.to_s)
+
+        if result.success?
+          render json: result.medical_shift, status: :ok
```

Comment:
we will extend these adjustments to old endpoints.
#destroy example.
in case of success:
- we will not return the object.
- return a json { message: "MedicalShift deleted successfully." }, status: :ok

- when the object cannot be found by id. Instead of raising "RecordNotFound", we will also return a json: 
{ message: "Object not found." }, status: :bad_request or :not_found

Adjust tests to reflect changes.

Reply:
my preference is to have `status :no_content`.

BTW, dont we have a standard approach for this at the api_controller?

Reply:
@edimossilva @danielnottingham With print bellow we can see the standard approach for RecordNotFound

![image](https://github.com/user-attachments/assets/76360216-a291-415d-8675-c80c761fa3dc)

Reply:
Dont you guys prefer to open a new issue to review all standards for delete endpoints?

Reply:
> Dont you guys prefer to open a new issue to review all standards for delete endpoints?

I can live with that, please create the issue.

Reply:
https://github.com/espoo-dev/med_system_backend/issues/207 In this issue we will correct this bevahior